src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,11 +103,9 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setPreferredWidth(20);
-			getTable().getColumnExt(1).setPreferredWidth(50);
-			getTable().getColumnExt(2).setMaxWidth(30);
-			getTable().getColumnExt(3).setPreferredWidth(20);
+			getTable().getColumnExt(0).setMaxWidth(70);
```

Comment:
OK.
It's possible to control the columns' auto resize behaviour with:
[JTable.setAutoResizeMode(int mode)](https://docs.oracle.com/javase/7/docs/api/javax/swing/JTable.html#setAutoResizeMode(int))

but does not seem to allow to ignore just one/first column.